sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -223,7 +200,7 @@ def fetch(self, qres, path=None, error_callback=None, **kwargs):
             name = url.split('/')[-1]
 
             # temporary fix !!! coz All QRBs have same start_time values
-            day = Time(qre.time.start.strftime('%Y-%m-%d')) + TimeDelta(i*u.day)
+            day = Time(qre['Time'].start.strftime('%Y-%m-%d')) + TimeDelta(i*u.day)
```

Comment:
What on earth is this comment?!

Reply:
Don't know what are `QRB` (may be queryresponse) here, but it probably mentions about the assumption that all file have `00:00` as start time values, which is used in `fetch` defined differently here to name the downloaded files.

Reply:
Can we remove this "temp" fix?

Reply:
I feel we should keep it, `SRSClient` is exceptional in terms of having its own `fetch()` method, so if there's generalization later done with `fetch`, this comment can be helpful.